inexor/ui/layer/InexorLayerManager.hpp

```diff
@@ -50,11 +50,11 @@ class InexorLayerManager : public inexor::ui::context::InexorContextProvider
         void SendMouseWheelEvent(const CefMouseEvent& event, int deltaX, int deltaY);
 
         // InexorContextProvider
-        void InitializeContext();
-        bool Execute(const CefString& name, CefRefPtr<CefV8Value> object, const CefV8ValueList& arguments, CefRefPtr<CefV8Value>& retval, CefString& exception);
-        bool Get(const CefString& name, const CefRefPtr<CefV8Value> object, CefRefPtr<CefV8Value>& retval, CefString& exception);
-        bool Set(const CefString& name, const CefRefPtr<CefV8Value> object, const CefRefPtr<CefV8Value> value, CefString& exception);
-        std::string GetContextName() { return "layer"; };
+        void InitializeContext() override;
+        bool Execute(const CefString& name, CefRefPtr<CefV8Value> object, const CefV8ValueList& arguments, CefRefPtr<CefV8Value>& retval, CefString& exception) override;
+        bool Get(const CefString& name, const CefRefPtr<CefV8Value> object, CefRefPtr<CefV8Value>& retval, CefString& exception) override;
+        bool Set(const CefString& name, const CefRefPtr<CefV8Value> object, const CefRefPtr<CefV8Value> value, CefString& exception) override;
+        std::string GetContextName() override { return "layer"; };
```

Comment:
Im not so sure about the "override" changes.
They seem to be incomplete.
I ran clang-tidy to do it in another branch and it resulted in way more override additions. (could be that I made some changes which required those additional overrides though)
Anyways, better than nothing, so just a note.

Reply:
I didn't notice more warnings, so I guess your work introduced them somehow.